src/emmetHelper.ts

```diff
@@ -540,11 +572,11 @@ function isExpandedTextNoise(syntax: string, abbreviation: string, expandedText:
 }
 
 /**
- * Returns options to be used by the @emmetio/expand-abbreviation module
```

Comment:
I'm pretty sure this was a misdocumentation that `The options to pass to the @emmetio/extract-abbreviation module` that's not true (because of this I was actually very confused what's going on). It actually returns the kindof unofficial "options" that this package uses hence I named it `HelperExpandOption` it has almost 80% overlap with `emmetio/extract-abbreviation` module's options but they are not same.